WMIAdventure/frontend/src/js/components/profile/molecules/UpgradeCardSection/UpgradeCardSection.js

```diff
@@ -133,14 +145,22 @@ class UpgradeCardSection extends React.Component {
                            setHeight={'auto'} setTop={'0'} setPosition={'static'}
                            setTranslateY={this.state.upgradeApprovePopUp.translateY}>
                         <UpgradeApprove cardName={this.props.cardName}
-                                        nextLevelCost={this.props.nextLevelCost}/>
+                                        nextLevelCost={this.props.nextLevelCost}
+                                        skillPoints={this.state.userPoints}
+                        />
                     </PopUp>
                 </TransparentBackground>
             );
         }
 
     }
 
+    canUpgrade() {
+        if (!this.state.userPoints) return true;

```

Comment:
Gdy `this.state.userPoints === 0` to otrzymamy czasami nieoczekiwane rezultaty, bo `0` będzie interpretowane jako `false`